client/Makefile

```diff
@@ -12,7 +12,14 @@ rebuild: clean all
 
 client-kv: client-kv.c
 	cd ../lib/marshal && make
-	$(CC) $(CFLAGS) client-kv.c -o $(OBJS) $(DEPS)
+	$(CC) $(CFLAGS) client-kv.c -o client-kv $(DEPS)
+
+client_api: client_api.c
+	$(CC) -c client_api.c -o client_api.o 
```

Comment:
`-o` flag is unnecessary (you're only compiling one field and it defaults to the name), but you can leave it